api_tests/nodes/serializers/test_serializers.py

```diff
@@ -109,28 +106,25 @@ def test_serialization(self):
             parse_date(attributes['date_registered']),
             reg.registered_date
         )
-        assert_equal(attributes['withdrawn'], reg.is_retracted)
+        assert attributes['withdrawn'] == reg.is_retracted
 
         # Relationships
         relationships = data['relationships']
         relationship_urls = {}
         for relationship in relationships:
             relationship_urls[relationship]=relationships[relationship]['links']['related']['href']
-        assert_in('registered_by', relationships)
+        assert 'registered_by' in relationships
         registered_by = relationships['registered_by']['links']['related']['href']
-        assert_equal(
-            urlparse(registered_by).path,
-            '/{}users/{}/'.format(API_BASE, user._id)
-        )
-        assert_in('registered_from', relationships)
+        assert (
+            urlparse(registered_by).path ==
+            '/{}users/{}/'.format(API_BASE, user._id))
+        assert 'registered_from' in relationships
         registered_from = relationships['registered_from']['links']['related']['href']
-        assert_equal(
-            urlparse(registered_from).path,
-            '/{}nodes/{}/'.format(API_BASE, reg.registered_from._id)
-        )
+        assert (
+            urlparse(registered_from).path ==
+            '/{}nodes/{}/'.format(API_BASE, reg.registered_from._id))
         for relationship in relationship_urls:
             if relationship in should_not_relate_to_registrations:
-                assert_not_in('/{}registrations/'.format(API_BASE), relationship_urls[relationship])
+                assert '/{}registrations/'.format(API_BASE) not in relationship_urls[relationship]
             else:
-                assert_in('/{}registrations/'.format(API_BASE), relationship_urls[relationship],
-                          'For key {}'.format(relationship))
+                assert '/{}registrations/'.format(API_BASE) in relationship_urls[relationship], 'For key {}'.format(relationship)
```

Comment:
This would be identical to the nose test correct?
assert '/{}registrations/'.format(API_BASE) in relationship_urls[relationship]
assert '/{}registrations/'.format(API_BASE) in 'For key {}'.format(relationship)